src/components/CreateAccount.jsx

```diff
@@ -1,9 +1,287 @@
-import React from 'react'
+import React from "react";
+import { useState, useEffect } from "react";
+import { Link, useNavigate } from "react-router-dom";
+import Grid from "@mui/material/Grid";
+import { Box, MenuItem, Typography } from "@mui/material";
+import Container from "@mui/material/Container";
+import {
+  FormControl,
+  InputLabel,
+  Input,
+  FormHelperText,
+  Button,
+  Select,
+} from "@mui/material";
+import { TextField } from "@mui/material";
+import CreateAccountImage from "../assets/create-account.jpg";
+import BrandLogo from "../assets/fiftyfive-logo.png";
+import MyButton from "../components/Button/MyButton";
+import { useForm } from "react-hook-form";
+import { createAccountPageStyles } from "../CreateAccount.styles.js";
+import { useToast } from '@chakra-ui/react'
 
 const CreateAccount = () => {
+  const { classes } = createAccountPageStyles();
+  
+  const [userAccountData, setUserAccountData] = useState({});
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm();
+
+  const navigate = useNavigate();
+
+  const onSubmit = async (data) => {
+    console.log(data);
+
+    let config = {
+        method: "post",
+        maxBodyLength: Infinity,
+        url: "http://192.168.1.34:8484/v1/signup",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        mode: "no-mode",
+        referrerPolicy: "no-referrer",
+        data: data,
+      };
+
+    await axios
+    .request(config)
+    .then((response) => {
+      console.log(response);
+      let accStatus = window.confirm("Account Created Successfully !");
+      if(accStatus){
+          navigate("/userlogin");
+      }
+    })
+    .catch((error) => {
+      console.log(error.message);
+    });
+  };
+
+  const axios = require("axios");
+  //   let data = JSON.stringify(userAccountData);
```

Comment:
same for commented code, commented code shouldn't be there unless keeping it for some reason, and in that case add comment why are you keeping it